src/NavigationActions.js

```diff
@@ -112,12 +112,15 @@ const completeTransition = createAction(COMPLETE_TRANSITION, payload => ({
 
 const openDrawer = createAction(OPEN_DRAWER, payload => ({
   type: OPEN_DRAWER,
+  suppressAnimation: false,
```

Comment:
I think better to have the same behavior in both test and real code.

Reply:
I'm not sure what behavior you're referring to here.